src/main/java/by/custom/utilcalculator/domain/tree/CommandTree.java

```diff
@@ -9,14 +10,13 @@
 import java.util.stream.Collectors;
 
 public class CommandTree {
-    private final Map<StepsIndicator, Command> fieldsToCommands;
-    private final Map<Command, List<StepsIndicator>> commandsToFields;
+    //private final Map<StepsIndicator, Command> fieldsToCommands;
```

Comment:
Add comments why it's commented

Reply:
In this case - we refused using maps, now we use nodes to build assotiations between COMMAND and STEP. Don't need maps anymore. Probably, I saved this maps for safety - if something go wrong in future with big branches, I can return maps easily. But it seems everything works fine - I guess, we can delete this maps.